Plugins: Improve wording around allow_loading_unsigned_plugins configuration (#35731)
grafana/grafana: docs/sources/administration/configuration.md

### allow_loading_unsigned_plugins

> Enter a comma-separated list of plugin identifiers to identify plugins that should be loaded even if they are unsigned. Plugins with modified signatures will never be loaded.

Reviewer: ```suggestion
Enter a comma-separated list of plugin identifiers to identify plugins to load even if they are unsigned. Plugins with modified signatures are never loaded.
```

> Please note that using this is _not_ recommended. For more information, refer to [Plugin signatures]({{< relref "../plugins/plugin-signatures.md" >}}).

Reviewer: ```suggestion
We do _not_ recommend using this option. For more information, refer to [Plugin signatures]({{< relref "../plugins/plugin-signatures.md" >}}).
```
